src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --colorBlye: #00acdc;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("/fonts/Roboto-Medium.ttf");
+  font-weight: 500;
+  font-style: normal;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 16px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__link {
+  margin: 10px 0;
+}
+
+.nav__list {
+  margin: 19px 0;
+  list-style-type: none;
+  display: flex;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  height: 60px;
+  font-size: 12px;
+  padding-top: 23px;
+  text-transform: uppercase;
+  transition: 3s;
```

Comment:
add transition-properties

Reply:
not fixed
https://github.com/mate-academy/layout_moyo-header/pull/3055/files#r1179473236